src/jupyter_starters/schema/v2.json

```diff
@@ -146,6 +146,16 @@
           "description": "[SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) string to use in Launcher cards and tab icons",
           "type": "string"
         },
+        "category": {
+          "title": "Category",
+          "description": "Category to place the starter card under in Launcher",
+          "type": "string"
```

Comment:
probably also needs a default

Reply:
Added it now in https://github.com/deathbeds/jupyterlab-starters/pull/78

But not 100% sure if it is really needed.